src/ape_run/_cli.py

```diff
@@ -114,14 +134,28 @@ def commands(self) -> Dict[str, Union[click.Command, click.Group]]:
         if not self._project.scripts_folder.exists():
             return {}
 
+        return self._get_cli_commands(self._project.scripts_folder)
+
+    def _get_cli_commands(self, base_path: Path) -> Dict:
         commands = {}
```

Comment:
I think this would fix the mypy issue.

(may have to import `Command`)

```suggestion
        commands: Dict[str, Command] = {} 
```